client/ping.go

```diff
@@ -2,13 +2,16 @@ package client // import "github.com/docker/docker/client"
 
 import (
 	"context"
+	"fmt"
 	"net/http"
 	"path"
 	"strings"
 
+	"github.com/docker/docker/api/server/httputils"
```

Comment:
We should probably avoid importing `api/server` in the client.


(I noticed we're already importing it in some test though, so we should evaluate if we can remove it there)

Reply:
Sounds good! I'm happy to need less changes.